test/e2e/specs/authentication/authentication__totp.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @group calypso-release
```

Comment:
These seem super clean and lightweight, should we just add it to `calypso-pr` instead?

Reply:
No, it can't be added to `calypso-pr` because of TOTP codes are limited to one time use, but the generation of the value is based on the time. 

This means that this spec cannot be run more than once in any 30-second window. You can test this for yourself by opening incognito window in two browsers, loading the login screen on both with details of a test account, then submitting the login form within the 30 second window shown by the TOTP generator. 

What happens is that despite entering the same code that Browser 1 used to log in, Calypso on Browser 2 will reject the code as it has been used already.

Reply:
Now, this won't be an issue if we manually set up 2FA each time, but that moves the race condition upstream. 

The best solution would be to create a test user each time and set up 2FA on it, but that would be quite costly database-wise.